src/tool/subcommands/snapshot_cmd.rs

```diff
@@ -44,6 +44,23 @@ pub enum SnapshotCommands {
         vendor: snapshot::TrustedVendor,
     },
 
+    /// Validates the snapshot.
+    ValidateDiffs {
+        /// Number of recent epochs to scan for broken links
+        #[arg(long, default_value_t = 2000)]
+        check_links: u32,
+        /// Assert the snapshot belongs to this network. If left blank, the
+        /// network will be inferred before executing messages.
+        #[arg(long)]
+        check_network: Option<crate::networks::NetworkChain>,
+        /// Number of recent epochs to scan for bad messages/transactions
+        #[arg(long, default_value_t = 60)]
+        check_stateroots: u32,
+        /// Path to a snapshot CAR, which may be zstd compressed
+        #[arg(required = true)]
+        snapshot_files: Vec<PathBuf>,
```

Comment:
Hopefully glob in bash should work for this `forest-tool snapshot validate-diffs *.car.zst`

Reply:
Seems Vec argument already accept multiple values.